metaswitch/common/logging_config.py

```diff
@@ -0,0 +1,57 @@
+# @file logging_config.py
+#
+# Project Clearwater - IMS in the Cloud
+# Copyright (C) 2015  Metaswitch Networks Ltd
```

Comment:
I think this should stay as 2013, because most of this code dates from 2013 and we conventionally leave copyright years as the earliest year (Mike left me a pull request comment to this effect once).